### PR TITLE
Make shortener not allowed host exception more explicit

### PR DESCRIPTION
**Pull request by asaunier**
Tested in a GMF 2.5 project. I can port it to master if this change is OK.

### CODE REVIEW

**Review by sbrunner — APPROVED** [review]

File: geoportal/c2cgeoportal_geoportal/views/shortener.py [hunk 1]
```diff
@@ -83,7 +83,12 @@ def create(self):
         uri_parts = urlparse(url)
         if "allowed_hosts" in self.settings:
             if uri_parts.netloc not in self.settings["allowed_hosts"]:  # pragma: no cover
-                raise HTTPBadRequest("The requested host is not allowed.")
+                raise HTTPBadRequest(
+                    "The requested host '{}' is not part of allowed hosts: '{}'".format(
+                        uri_parts.netloc,
+                        "', '".join(self.settings["allowed_hosts"])
```
Comment (sbrunner) [inline geoportal/c2cgeoportal_geoportal/views/shortener.py:89]: Usually, we do like this:
```suggestion
                    "The requested host '{}' is not part of allowed hosts: {}".format(
                        uri_parts.netloc,
                        ", ".join(self.settings["allowed_hosts"])
```

**Comment by asaunier** [comment]
@sbrunner Thanks! I have integrated your suggestion. I will press the "squash and merge" button as soon as the CI passes.

**Review by sbrunner — APPROVED** [review]

**Comment by asaunier** [comment]
@sbrunner With the "backport" labels, if I merge the current PR (against 2.5) it will "backport" the change to 2.6 and master as well, right?

**Comment by sbrunner** [comment]
Yes :-)

**Pull request merged**